static/resources/sharedworker.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Mozilla Corporation
```

Comment:
I'll wait until I figure it out about the license: https://github.com/foolip/mdn-bcd-collector/pull/548#discussion_r495488389

Or should this still be Google LLC as this is really just a copy of worker.js? 👀

Reply:
Personally I would think that the license would remain the same since the new file is a carbon copy of the existing file?  (I'm requesting a revert of this change because it's modifying the service worker file, not the new shared worked.  😛)

Reply:
Okay, I'll do that 👍